cinema/views.py

```diff
@@ -1,45 +1,112 @@
-from rest_framework.decorators import api_view
+from urllib.request import Request
```

Comment:
The import of `Request` from `urllib.request` seems unnecessary as it is not used in the code. Consider removing it to clean up the imports.